src/treeview_switches.ml

```diff
@@ -33,10 +33,7 @@ module Dependency = struct
 
   let context_value = function
     | Switch _ -> "opam-switch"
-    | Package dep -> (
-      match Opam.Package.documentation dep with
-      | Some _ -> "package-with-doc"
-      | None -> "package")
+    | Package _ -> "package"
```

Comment:
```suggestion
    | Package _ -> "opam-package"
```

Reply:
Should we make the same change in `treeview-sandbox` ?

https://github.com/ocamllabs/vscode-ocaml-platform/blob/9bfe28d361668f6a360d1da626576e1b3c3e754b/src/treeview_sandbox.ml#L36

Reply:
Maybe? I don't have a strong opinion about this change 🙂

Reply:
[`98e76f9` (#1331)](https://github.com/ocamllabs/vscode-ocaml-platform/pull/1331/commits/98e76f9e59970027a79924c9cee8c3f5327490ee)